apps/backend/src/reporting/reporting.service.ts

```diff
@@ -696,7 +696,7 @@ export class ReportingService {
     qb.select('payment.status');
     qb.addSelect('COUNT(payment.status)', 'count');
     qb.leftJoin('payment.payment_method', 'payment_method');
-    qb.andWhere('payment_method.classification = :classification', {
+    qb.where('payment_method.classification = :classification', {
```

Comment:
This is fine, but FYI, `andWhere` acts like a `where` if its the only clause, so it makes it like 0.05 seconds shorter to add further clauses in the future.

No need to change :p